src/Compilers/Core/MSBuildTask/ValidateBootstrap.cs

```diff
@@ -72,14 +72,7 @@ public override bool Execute()
                 }
             }
 
-            // The number chosen is arbitrary here.  The goal of this check is to catch cases where a coding error has 
-            // broken our ability to use the compiler server in the bootstrap phase.
-            //
-            // It's possible on completely correct code for the server connection to fail.  There could be simply 
-            // named pipe errors, CPU load causing timeouts, etc ...  Hence flagging a single failure would produce
-            // a lot of false positives.  The current value was chosen as a reasonable number for warranting an 
-            // investigation.
-            const int maxRejectCount = 5;
+            const int maxRejectCount = 0;
```

Comment:
Perhaps modify the existing comment, rather than removing it? Otherwise it appears that we just have a useless bit of logic for retrying rejections.